src/sentry/api/api_owners.py

```diff
@@ -28,3 +28,4 @@ class ApiOwner(Enum):
     TELEMETRY_EXPERIENCE = "telemetry-experience"
     UNOWNED = "unowned"
     WEB_FRONTEND_SDKS = "team-web-sdk-frontend"
+    REMOTE_CONFIG = "replay-backend"
```

Comment:
replay-backend is already here in the enum

Reply:
It will be replaced when the remote-config team is created.  This shouldn't affect anything.